GravityPDF/gravity-pdf
# Remove Always Save PDF setting

## Description

As this was used to ensure the gfpdf_post_save_pdf hook was consistently triggered, we can just check to see if any filters have been attached to this hook during runtime and simplify the UI.

## Testing instructions
1. Create a new PDF on a form
2. Add the hook https://docs.gravitypdf.com/v5/gfpdf_post_save_pdf to your code and ensure it's setup to run on the form you just created the PDF for
3. Submit the Gravity Form so a new entry is created
4. Check that the PDF has been moved to the /PDFs/ directory at the root of your WordPress installation

## Screenshots 

## Checklist:
